resources/tag_models/event.json

```diff
@@ -0,0 +1,29 @@
+{
+  "tag": {
+	"api_key": "", // String - API key - Make sure the request is from a valid client (idea nicked from Keen.io) useful if a page gets copied onto a Russian website and creates noise
+	"version": "", // Version of the tracking client e.g. "JS v1.2"
+	"type": "event", // Type of request (page or event)
+	"id": "", // ID of this request, must be unique
```

Comment:
> Without a hierarchy we know none of the above.

I think you guys have blinkers on for this hierarchy thing. I have been running the phrase "event -> event hierarchy" over in my head for about 5 minutes and I have not the slightest clue what it means.  I know it's frustrating to you, especially when you talk to lots of different people, so maybe it's worth having a larger gathering to discuss this point?